metametamodel/metametamodel.adoc

```diff
@@ -6,7 +6,7 @@ include::../shared/issue-footnotes.adoc[]
 :sectnums:
 
 
-In this document we describe the Meta-Metamodel to be used by LionWeb.
+In this document we describe the Meta-Metamodel to be used by LionWeb *version 2024.1*.
```

Comment:
Should we then rename this file? Should we keep side by side a version of the document describing version 2023.1?

Reply:
There is a fixed version for 2023.1 in the folder with the same name. This is the _current_ version.

Reply:
Makes sense